src/Core/Services/IFeatureService.cs

```diff
@@ -1,5 +1,11 @@
 ﻿namespace Bit.Core.Services;
 
+public struct FeatureFlagContext
```

Comment:
🎨 I am still hung up on this being associated to feature flags. My opinion is that context is a platform-level concept and should be there and not in these classes.

Reply:
Good thought, I was thinking of context in terms of the LD context, but that's entirely derived from `CurrentContext`. A safe report of that context should fully describe anything that any given Feature Flag provider could use to key off of